lib/decidim/github_manager/querier.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+require "faraday"
+require "json"
+
+module Decidim
+  module GithubManager
+    #
+    # Allows to make GET requests to GitHub about Issues and Pull Requests
+    #
+    class Querier
+      def initialize(token:, issue_id:)
+        @token = token
+        @issue_id = issue_id
+      end
+
+      #
+      # Makes the GET request and parses the response of an Issue or Pull Request in GitHub
+      #
+      # @return [Hash]
+      #
+      def call
+        parse(issue_metadata)
```

Comment:
This will fail in case the pull request ID did not exist.

I would maybe add here some check, such as:
```suggestion
        data = issue_metadata
        return unless data["number"]

        parse(issue_metadata)
```